app/native.js

```diff
@@ -3,12 +3,11 @@
 import { AppRegistry } from 'react-native';
 import {
   SingleHotelStandalonePackage,
-  HotelsStandalonePackage,
   NewHotelsStandAlonePackage,
 } from '@kiwicom/react-native-app-hotels';
 
 // Hotels
-AppRegistry.registerComponent('KiwiHotels', () => HotelsStandalonePackage);
+AppRegistry.registerComponent('KiwiHotels', () => NewHotelsStandAlonePackage); // TODO: Remove later
```

Comment:
I think that makes sense yes, after thinking about it, I am not sure we should even register `KiwiHotels` any more, since it has other props requirements